src/Microsoft.AspNetCore.Razor/CodeGenerators/GeneratedTagHelperContext.cs

```diff
@@ -38,6 +38,8 @@ public GeneratedTagHelperContext()
             TagHelperOutputIsContentModifiedPropertyName = "IsContentModified";
             TagHelperOutputContentPropertyName = "Content";
             TagHelperOutputGetChildContentAsyncMethodName = "GetChildContentAsync";
+            TagHelperAttributeTypeName = "Microsoft.AspNetCore.Razor.TagHelpers.TagHelperAttribute";
+            HtmlStringTypeName = "Microsoft.AspNetCore.Mvc.Rendering.HtmlString";
```

Comment:
What difference does `Html.Raw()` make? `HtmlString` is a subclass of `HtmlEncodedString` that exists only for back-compat.